engine/engine/src/test/wscript

```diff
@@ -1,9 +1,31 @@
 #! /usr/bin/env python
 
-import waflib.Task, waflib.TaskGen, waflib.Options
+import waflib.Task, waflib.TaskGen
+from waflib import Options
 import os, sys, glob, re
+import run, sdk
 import waf_gamesys
 
+# 1. We want to run it with run.py so that it logs nicely with [exec] ...
+# 2. We need to add DYLD_INSERT_LIBRARIES for macOS
+def run_bob(task):
```

Comment:
It was surprisingly hard to insert this environment variable, given all the layers we use to invoke commands.

This works, and we might later on move it to some helper function and/or feature.